README.md

```diff
@@ -46,3 +46,76 @@ to acknowledge contributors and describe the publication record that is created
 by archiving it using [zenodo.org](https://zenodo.org/). You may also want to
 consider acknowledging contributors with the
 [allcontributors bot](https://allcontributors.org/docs/en/bot/overview).
+
+
+## Dataverse docker for running tests
+
+The [dataverse-docker](https://github.com/IQSS/dataverse-docker) repository 
+provides everything needed to run dataverse in a container. The continuous 
+integration test build is based on that. The setup script used for the 
+respective AppVeyor build is under `tools/ci/setup_docker_dataverse`.
+
+You can use this docker setup locally on your machine, too, provided it's 
+running Linux and you have `docker` and `docker-compose` installed. All 
+involved scripts (from our end as well as the dataverse-docker repo) are 
+completely ignorant of Windows.
+The basic setup is this:
+
+    git clone https://github.com/IQSS/dataverse-docker
+    cd dataverse-docker
+    export traefikhost=localhost
+    docker network create traefik
+    cp .env_sample .env
+
+If you want to customize your setup, you may want to edit this `.env` file.
+Note, that the following call to `docker-compose` relies on being in the 
+directory of the docker-compose file and that `.env`. If that's not suitable 
+in your case, `docker-compose` provides a `--file` and a `--env-file` parameter.
+
+    docker-compose up -d
+
+This should give you a running server at `http://localhost:8080`. Note, that 
+it may take a moment for the server to come up. If you go to that address in 
+a browser you should be abe to log in as `dataverseAdmin` with password 
+`admin` (which you will instantly be required to change). This allows to 
+create dataverses, datasets, files, etc. An API token for that user is to be 
+found under that user's menu (upper right).
+
```

Comment:
You demoed very helpful docker rm commands, could we add those into the README?

Reply:
Will do